apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -922,16 +923,18 @@ export function renderConnectYourDevice({
 export function renderLoading({
   t,
   description,
+  lockModal = false,
```

Comment:
As a precaution I would have made the default to true and turned it off for specific cases where we're sure it doesn't break, it would be enough to cover the scope of the ticket.
But seeing that you've made changes in other specific cases like the swap confirmation modal, I trust you to have tested all usages of this 👍

Reply:
I trust me too